dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -101,7 +101,7 @@ RUN python3 -m pip install aiohttp \
                  scapy==2.4.5 \
                  setuptools-rust \
                  six \
-                 snappi[ixnetwork,convergence]==0.11.16 \
+                 snappi[ixnetwork,convergence]==0.9.1 \
```

Comment:
on line 355, it's also python3, please keep the same with line 104.

For the case that python2 must be different from python3, to avoid future confusion could you leave a comment on the python2(around line 185) indicating this?

Reply:
understood, it's done